plugins/wacom-usb/fu-wac-device.c

```diff
@@ -15,6 +15,7 @@
 #include "fu-wac-module-bluetooth-id9.h"
 #include "fu-wac-module-bluetooth.h"
 #include "fu-wac-module-scaler.h"
+#include "fu-wac-module-sub-cpu.h"
```

Comment:
I'm not sure I like the name -- is sub_cpu what you use internally? Not something like mcu?

Reply:
The internal name is indeed "sub cpu". Our team uses "mcu" exclusively for the micro running the "main" firmware.